awesome/CONTRIBUTING.md

```diff
@@ -70,6 +76,32 @@ Now, you are ready to submit a PR for your changes. Please make sure you have go
 
 TBD
 
+#### STEP 4. Resolve conflicts
```

Comment:
You can use GPT or Bard to help you refine the text.

Reply:
done